src/main/java/kitchenpos/menu/domain/Price.java

```diff
@@ -0,0 +1,49 @@
+package kitchenpos.menu.domain;
+
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import java.math.BigDecimal;
+import java.util.Objects;
+
+@Embeddable
+public class Price {
+    private static final String EXCEPTION_MESSAGE_PRICE_IS_NOT_NEGATIVE = "가격은 음수일 수 없습니다.";
+    private static final int ZERO = 0;
+    @Column(nullable = false, precision = 19, scale = 2)
+    private BigDecimal price = BigDecimal.ZERO;
+
+    protected Price() {
+    }
+
+    public Price(BigDecimal price) {
+        validatePrice(price);
+        this.price = price;
+    }
+
+    public BigDecimal value() {
+        return price;
+    }
+
+    private void validatePrice(BigDecimal price) {
+        if (Objects.isNull(price) || price.compareTo(BigDecimal.ZERO) < ZERO) {
```

Comment:
Objects.isNull 은 obj == null 만 검사해서 가독성에 큰 도움은 안되는듯 싶습니다.
javadoc에도 Predicate 같은 곳에 사용하기 위해 만들었다고 나와있어요.

```
Returns true if the provided reference is null otherwise returns false.
Params: obj – a reference to be checked against null
Returns: true if the provided reference is null otherwise false
API Note: This method exists to be used as a java.util.function.Predicate, filter(Objects::isNull)
Since: 1.8
See Also: java.util.function.Predicate
```

Reply:
javadoc 을 읽는 습관을 들여야겠네요..!
가독성 측면에서 해당 부분을 `null == price` 등으로 개선해보겠습니다!